public/catalogue-frontend.css

```diff
@@ -12286,4 +12269,60 @@ table.shutter-events {
 
 #shutter-events-form #serviceName {
   margin-bottom: 10px;
+}
+
+#environmentTabs li a {
+  padding: 2px;
+  padding-bottom: 0px;
+}
+
+#environmentTabs li a .board {
+  margin-bottom: 2px;
+  border: none;
+}
+
+#environmentTabs li a .board .board__heading {
+  border: none;
+  padding: 6px;
+  padding-top: 2px;
+  padding-bottom: 0px;
+}
+
+#environmentTabs li a .board .board__heading p {
+  margin-bottom: 0px;
+}
+
+#environmentTabs li a .board .board__body {
+  padding: 0px;
+}
+
+.nav-tabs > li.active > a > .board > .board__heading {
+  background: inherit;
+}
+
+.environment-panes {
+  padding: 16px;
+  padding-bottom: 0px;
+  margin-bottom: 16px;
+  border: 1px solid #ddd;
+  border-top: none;
+}
```

Comment:
I would like to suggest these changes. I think the CSS needs more work than this though, at some point in time.

```
#environmentTabs li a .board .board__heading {
   border: none;
-  padding: 6px;
   padding-top: 2px;
-  padding-bottom: 0px;
 }

-.nav-tabs > li.active > a > .board > .board__heading {
+.nav-tabs > li > a > .board > .board__heading {
   background: inherit;
 }

+.environment-panes .board {
+  border: none;
+}

```